com.unity.netcode.gameobjects/Editor/NetworkObjectEditor.cs

```diff
@@ -95,7 +97,11 @@ public override void OnInspectorGUI()
             }
             else
             {
-                base.OnInspectorGUI();
+                EditorGUI.BeginChangeCheck();
```

Comment:
Duplicated code... apparently one cannot add a class in between NetworkObjectEditor and Unity.Editor in the hierarchy without triggering a "public api change" error in the validator, so instead of being able to reuse this code, I have to copy/paste it...

That's pretty silly IMO.